app/src/main/java/com/sjh/sunflower_sksowk156/feature/main/navigation/MainNavigation.kt

```diff
@@ -0,0 +1,17 @@
+package com.sjh.sunflower_sksowk156.feature.main.navigation
+
+import androidx.compose.ui.Modifier
+import androidx.navigation.NavGraphBuilder
+import androidx.navigation.compose.composable
+import com.sjh.sunflower_sksowk156.feature.main.MainScreen
+
+const val MAIN_ROUTE = "/main"
+
+fun NavGraphBuilder.mainScreen(
+    modifier: Modifier,
+    onPlantItemClick: (String) -> Unit,
```

Comment:
```suggestion
    onClickPlantItem: (String) -> Unit,
```
string의 parameter name이 있다면 무엇이 전달되는지 조금 더 이해하기 쉬울 것 같습니다.